src/wizards/tokenSwapDealWizard/components/tokenDetails/tokenDetails.ts

```diff
@@ -0,0 +1,122 @@
+import { bindable } from "aurelia-typed-observable-plugin";
+import "./tokenDetails.scss";
+import { IToken } from "../../../../entities/DealRegistrationTokenSwap";
+import { autoinject, bindingMode } from "aurelia-framework";
+import { AureliaHelperService } from "../../../../services/AureliaHelperService";
+import { TokenService } from "../../../../services/TokenService";
+import {
+  validateTrigger,
+  ValidationController,
+  ValidationControllerFactory,
+  ValidationRules,
+} from "aurelia-validation";
+import { PrimeRenderer } from "../../../../resources/elements/primeDesignSystem/validation/primeRenderer";
+import { Validation } from "../../../../services/ValidationService";
+import { Utils } from "../../../../services/utils";
+
+@autoinject
+export class TokenDetails {
+  @bindable token: IToken;
+  @bindable.string() viewMode: "edit" | "view" = "edit";
```

Comment:
Is `viewMode` used as a bindable?
Couldn't find it being used in `tokenDetailsStage.html` in `<token-details>`